src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  margin: 20px 0;
+}
+
+.input {
+  width: 100%;
```

Comment:
tests fail because of wrong font-family
it is overwritten by default browser styles:

![image](https://user-images.githubusercontent.com/86923089/223785050-3d3e122b-4379-48e4-8d7f-2843e57b2384.png)

set `font-family: inherit` here, and for `::placeholder` to set inherited values by hand and avoid browser overwriting